test/api/v01/vrp_test.rb

```diff
@@ -181,7 +186,6 @@ def test_cannot_get_vrp
 
     get "/0.1/vrp/jobs/#{@job_id}", api_key: 'vroom'
     assert_equal 404, last_response.status, last_response.body
-    assert_equal JSON.parse(last_response.body)['status'], 'Not Found'
```

Comment:
Status is not used anymore.

Reply:
status has been replaced, you should not remove this line, but replace it